starter_code/package.json

```diff
@@ -4,14 +4,15 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "start": "node app.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon -e hbs --inspect app.js"
```

Comment:
i see nodemon not in ur dependencies by this way if I have no nodemon globally installed it will break for me.
Pls add nodemon in ur dev dependencies